latest_cov_build.py

```diff
@@ -70,3 +71,14 @@
         tar.extractall(path='tools')
 
 os.remove(name)
+
+# Download gcno archive
```

Comment:
Let's move this code close to where we download the build (since these files are closely related with the build).